win32ss/gdi/ntgdi/freetype.c

```diff
@@ -426,6 +426,9 @@ VOID DumpPrivateFontList(BOOL bDoLock)
 {
     PPROCESSINFO Win32Process = PsGetCurrentProcessWin32Process();
 
+    if (!Win32Process)
```

Comment:
Add a `DPRINT,` as in `DumpFontGDI`?
Is this worth a comment to explain when this case is expected to happen?

Reply:
When Win32Process becomes NULL?